components/scream/src/physics/p3/p3_f90.cpp

```diff
@@ -54,6 +55,7 @@ FortranData::FortranData (Int ncol_, Int nlev_)
   dzq = Array2("vertical grid spacing, m", ncol, nlev);
   npccn = Array2("ccn activated number tendency, kg-1 s-1", ncol, nlev);
   naai = Array2("activated nuclei concentration, kg-1", ncol, nlev);
+  qc_relvar = Array2("Assumed SGS 1/(var(qc)/mean(qc)), kg2/kg2", ncol, nlev);
```

Comment:
It doesn't look like we're using this convention in this file, but often it's helpful to surround the units with square brackets.

Reply:
Yeah, I noticed this too. I'd prefer to put all units in square brackets but didn't want to add that to this already big PR. I think we will need a big "final cleanup" PR once we're done. I've created a confluence page to start a list of things we want to change: https://acme-climate.atlassian.net/wiki/spaces/NGDNA/pages/1465483352/Stuff+to+include+in+Final+Cleanup+PR